internal/pkg/version/check_remote.go

```diff
@@ -120,6 +122,16 @@ func (c *checker) getLatestVersion() (string, error) {
 			continue
 		}
 
+		// Skip draft
+		if release["draft"] == true {
+			continue
+		}
+
+		// Skip pre-release
+		if release["prerelease"] == true {
+			continue
+		}
+
```

Comment:
Pre-release a draft nebudu zobrazovat version warning.
To nam umozni urobit release ako draft, a ked je vsekto OK (napr. mergnuty winget/....), tak to jednoducho v GitHub-e prepnut.